masterlist.yaml

```
@@ -7919,7 +7919,7 @@ plugins:
         util: 'SSEEdit v4.0.2f'
 
   - name: 'Requiem.esp'
-    url: [ 'https://www.nexusmods.com/skyrim/mods/19281/' ]
+    url: [ 'https://www.nexusmods.com/skyrim/mods/60888/' ]
```

@Kaenguru-Knobel-Kette 

```suggestion
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/60888/' ]
```